src/layouts/partials/HeaderPartial/HeaderPartial.jsx

```diff
@@ -432,7 +430,7 @@ class HeaderPartial extends PureComponent {
     )
   }
 
-  handleNotificationsOpen (e) {
+  handleNotificationsOpen = (e) => {
```

Comment:
move handler before render

Reply:
Resort of class members is massive refactoring and shuld done in separate task